compiler/llvm/llvmGlobalToWide.cpp

```diff
@@ -1037,10 +1037,17 @@ namespace {
 
       if (newSrcTy != srcTy || newResTy != resTy) {
         // gather the indices
+#if HAVE_LLVM_VER >= 130
         SmallVector<Constant*> idxList;
```

Comment:
Is leaving out the initial size of the `SmallVector` intentional here? Could it just be declared outside of the macro guarded area?

Separately, it seems we could just use the old-style begin and end iterator regardless of LLVM version, to avoid another split based on version, but that's personal preference.